enterprise/configuration-files/nginx.conf.template

```diff
@@ -0,0 +1,30 @@
+events {
+    multi_accept on;
+    use epoll;
+}
+
+http {
+    proxy_cache off;
+
+    upstream backend {
+        server ${UPSTREAM_SERVER};
+    }
+
+    server {
```

Comment:
why do we need the separate file above if the server configuration is in here as well?